internal/static/host.go

```diff
@@ -0,0 +1,47 @@
+package static
+
+import (
+	"io"
+	"log"
+	"net"
+)
+
+// InitializeHost is a temporary helper to simulate an upstream host that will print off any incoming data.
+func InitializeHost(tcpNetwork, address string) net.Listener {
+	h, err := net.Listen(tcpNetwork, address)
+	if err != nil {
+		log.Fatalf("unable to listen on %s: %s", address, err)
+	}
+
+	log.Printf("Statically defined host listening on %s", h.Addr())
+
+	go func() {
+		// In a goroutine, continuously accept connections.
+		for {
+
+			// TODO: In the PR which includes health checks, set up some interval or predictable/controllable behavior where a host will not accept connection so that we can test the health checking process.
+			conn, err := h.Accept()
+			if err != nil {
+				log.Printf("host was unable to accept incoming connection: %s", err)
+				continue
```

Comment:
With the static hosts, the behavior is less critical. So I didn't spend as much time on this part of the code.

During the next PR with health checks, we'll actually configure some of these to fail to accept connection occasionally, and at that point we'll probably still want the hosts running after a failed connection so that health checker can re-add hosts which recover health.

I'll add a comment about that.

Reply:
Comment added.